example/full/table.properties

```diff
@@ -74,17 +74,17 @@ sleeper.table.gc.delay.minutes=15
 # by the whole instance (set in the instance properties).
 sleeper.table.compaction.strategy.class=sleeper.compaction.strategy.impl.SizeRatioCompactionStrategy
 
-# The number of files to read in a compaction job. Note that the state store must support atomic
-# updates for this many files.
+# The maximum number of files to read in a compaction job. Note that the state store must support
+# atomic updates for this many files.
 # The DynamoDBStateStore must be able to atomically apply 2 updates for each input file to remove the
 # file references and update the file reference count, and another 2 updates for an output file to add
 # a new file reference and update the reference count. There's a limit of 100 atomic updates, which
 # equates to 49 files in a compaction.
 # See also: https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/transaction-apis.html.
-# Also note that as this many files may need to be open simultaneously, the value of
-# 'sleeper.fs.s3a.max-connections' must be at least the value of this plus one (the extra one is for
-# the output file).
-sleeper.table.compaction.files.batch.size=49
+# Also note that as this many files may need to be open simultaneously. The value of
```

Comment:
Should there be a full stop here? It reads as if the previous sentence should carry on. You could also swap these around: `Also note that the value of 'sleeper.fs.s3a.max-connections' must be at least the value of this plus one, as this many files need to be open simultaneously.`

Reply:
Ah, there's an extra "as" here. I've removed it, thanks.